spec/features/user_deletes_account_spec.rb

```diff
@@ -0,0 +1,30 @@
+require 'rails_helper'
+
+feature 'user deletes account' do
+  scenario 'successfully deletes account' do
+    user = FactoryGirl.create(:user)
+
+    visit root_path
+    sign_in(user)
+
+    click_link 'Edit Account'
+    click_button 'Delete My Account'
+
+    expect(page).to have_content(
+      'Bye! Your account has been successfully cancelled.'
+    )
+
+    sign_in(user)
+
+    expect(page).to have_content('Invalid email or password.')
+    expect(page).to have_content('Sign Up Here!')
+    expect(page).to_not have_content("Welcome Back!")
+  end
+end
+
+def sign_in(user)
```

Comment:
There is a way to do this so that all of your specs can share it. 

Check out this for more: 

https://www.relishapp.com/rspec/rspec-core/v/3-4/docs/helper-methods/define-helper-methods-in-a-module